docs/source/package/tools.rst

```diff
@@ -0,0 +1,15 @@
+================
+Additional Tools
+================
+
+.. toctree::
+
+   tools/tools
```

Comment:
I think we can plan to have the stuff in `sopel.tools` documented here, without the redundant file path?

Personally I'm not a huge fan of having too many pages that contain only a TOC of subpages.

Reply:
Yeah I'm not very happy with that one but it's a lot of things put in the wrong places and everything is more complicated than it should be.

Reply:
I reverted that back so there is no tools/tools anymore.